docs/language_versions.md

```diff
@@ -3,6 +3,9 @@
 For review, nothing set in sand yet (and these things should never be set in
 stone; we need to keep learning and adapting them over time).
 
+(For details on adding new versions or features in new versions,
+see [New language versions](./New-language-versions).)
```

Comment:
This link path doesn't match the name of the new file (`versions` versus `features`).  Also worth making sure the filename matches the surrounding naming conventions (case, kebab, etc.).